### PR TITLE
pages*: improve list placeholders and wording of descriptions

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/buzzphrase.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 `buzzphrase`
 
-- Output a phrase formatted as [i]mperative verb + past tense [v]erb + [a]djective + plural [N]oun:
+- Print a phrase formatted as [i]mperative verb + past tense [v]erb + [a]djective + plural [N]oun:
 
 `buzzphrase {{'{i} {v} {a} {N}'}}`
 
-- Output 4 phrases formatted as present participle [V]erb + [a]djective + singular [n]oun + [f]inal:
+- Print `n` phrases formatted as present participle [V]erb + [a]djective + singular [n]oun + [f]inal:
```
Comment (gutjuri) [inline pages/common/buzzphrase.md:14]:
```suggestion
- Print `k` phrases formatted as present participle [V]erb + [a]djective + singular [n]oun + [f]inal:
```

`n` is already used for [n]oun later on

File: pages/common/buzzphrase.md [hunk 2]
```diff
@@ -7,10 +7,10 @@
...
 
-`buzzphrase {{4 '{V} {a} {n} {f}'}}`
+`buzzphrase {{4}} {{'{V} {a} {n} {f}'}}`
```
Comment (gutjuri) [inline pages/common/buzzphrase.md:16]:
```suggestion
`buzzphrase {{k}} {{'{V} {a} {n} {f}'}}`
```

File: pages/common/clamav.md [hunk 3]
```diff
@@ -4,14 +4,14 @@
 > ClamAV isn't a command, but a set of commands.
 > More information: <https://www.clamav.net>.
 
-- Show the tldr page for scan files using the `clamd` daemon:
+- View documentation for scan files using the `clamd` daemon:
```
Comment (gutjuri) [inline pages/common/clamav.md:7]:
```suggestion
- View documentation for scanning files using the `clamd` daemon:
```

File: pages/common/clamav.md [hunk 4]
```diff
@@ -4,14 +4,14 @@
...
 
 `tldr clamdscan`
 
-- Show the tldr page for scan files without the `clamd` daemon running:
+- View documentation for scan files without the `clamd` daemon running:
```
Comment (gutjuri) [inline pages/common/clamav.md:11]:
```suggestion
- View documentation for scanning files without the `clamd` daemon running:
```

**Review by gutjuri — APPROVED** [review]

**Comment by gutjuri** [comment]
Thanks! I have added some suggestions.

File: pages/common/adguardhome.md [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 `AdGuardHome`
 
-- Run AdGuard Home with a specific config:
+- Specify a different config file:
```
[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages/common/adguardhome.md:10]: What do you think of non-default?

File: pages/common/adguardhome.md [hunk 6]
```diff
@@ -7,7 +7,7 @@
 
 `AdGuardHome`
 
-- Run AdGuard Home with a specific config:
+- Specify a non-default config file:
```
Comment (kbdharun) [inline pages/common/adguardhome.md:10]:
```suggestion
- Specify a non-default configuration file:
```

File: pages/common/goimports.md [hunk 7]
```diff
@@ -5,16 +5,16 @@
...
 
 - Display diffs and write the result back to the source file:
 
-`goimports -w -d {{path/to/file}}.go`
+`goimports -w -d {{path/to/file.go}}`
 
 - Set the import prefix string after 3rd-party packages (comma-separated list):
 
-`goimports -local {{path/to/package}} {{path/to/file}}.go`
+`goimports -local {{path/to/package1,path/to/package2,...}} {{path/to/file}}.go`
```
Comment (kbdharun) [inline pages/common/goimports.md:20]:
```suggestion
`goimports -local {{path/to/package1,path/to/package2,...}} {{path/to/file.go}}`
```

File: pages/linux/module.md [hunk 8]
```diff
@@ -29,4 +29,4 @@
 
 - Specify user-created modules:
 
-`module use {{path/to/modulefiles}}`
+`module use {{path/to/module_file1 path/to/modulefile2 ...}}`
```
Comment (kbdharun) [inline pages/linux/module.md:32]:
```suggestion
`module use {{path/to/module_file1 path/to/module_file2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some minor suggestions for this page.

**Comment by vitorhcl** [comment]
For the sake of consistency, when the separator is a comma, should we also use "... one or more things", not using "comma-separated list" in these cases, given that the placeholder already shows it?

**Comment by kbdharun** [comment]
> For the sake of consistency, when the separator is a comma, should we also use "... one or more things", not using "comma-separated list" in these cases, given that the placeholder already shows it?

That would work too, let's see what others think.

**Review by kbdharun — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages/common/adguardhome.md [hunk 9]
```diff
@@ -7,7 +7,7 @@
 
 `AdGuardHome`
 
-- Run AdGuard Home with a specific config:
+- Specify a non-default configuration file:
```
Comment (acuteenvy) [inline pages/common/adguardhome.md:10]:
```suggestion
- Specify a configuration file:
```

File: pages/common/bgpgrep.md [hunk 10]
```diff
@@ -4,26 +4,26 @@
 > Can read files compressed with gzip, bzip2 and xz.
 > More information: <https://codeberg.org/1414codeforge/ubgpsuite>.
 
-- Output all routes:
+- List all routes:
 
 `bgpgrep {{master6.mrt}}`
 
-- Output all routes received from a specific peer, determined by the peer's AS number:
+- List all routes received from a specific peer, determined by the peer's AS number:
```
Comment (acuteenvy) [inline pages/common/bgpgrep.md:11]:
```suggestion
- List routes received from a specific peer, determined by the peer's AS number:
```

File: pages/common/bgpgrep.md [hunk 11]
```diff
@@ -4,26 +4,26 @@
...
 
 `bgpgrep {{master4.mrt}} -peer {{64498}}`
 
-- Output all routes received from a specific peer, determined by the peer's IP address:
+- List all routes received from a specific peer, determined by the peer's IP address:
```
Comment (acuteenvy) [inline pages/common/bgpgrep.md:15]:
```suggestion
- List routes received from a specific peer, determined by the peer's IP address:
```

File: pages/common/bgpgrep.md [hunk 12]
```diff
@@ -4,26 +4,26 @@
...
 
 `bgpgrep {{master4.mrt.bz2}} -peer {{2001:db8:dead:cafe:acd::19e}}`
 
-- Output all routes which have certain ASNs in their AS path:
+- List all routes which have certain ASNs in their AS path:
```
Comment (acuteenvy) [inline pages/common/bgpgrep.md:19]:
```suggestion
- List routes which have certain ASNs in their AS path:
```

File: pages/common/bgpgrep.md [hunk 13]
```diff
@@ -4,26 +4,26 @@
...
 
 `bgpgrep {{master6.mrt.bz2}} -aspath '{{64498 64510}}'`
 
-- Output all routes that lead to a specific address:
+- List all routes that lead to a specific address:
```
Comment (acuteenvy) [inline pages/common/bgpgrep.md:23]:
```suggestion
- List routes that lead to a specific address:
```

File: pages/common/bgpgrep.md [hunk 14]
```diff
@@ -4,26 +4,26 @@
...
 
 `bgpgrep {{master6.mrt.bz2}} -supernet '{{2001:db8:dead:cafe:aef::5}}'`
 
-- Output all routes that have communities from a specific AS:
+- List all routes that have communities from a specific AS:
```
Comment (acuteenvy) [inline pages/common/bgpgrep.md:27]:
```suggestion
- List routes that have communities from a specific AS:
```

File: pages/common/docker-network.md [hunk 15]
```diff
@@ -11,9 +11,9 @@
 
 `docker network create --driver {{driver_name}} {{network_name}}`
 
-- Display detailed information of a space-separated list of networks:
+- Display detailed information of one or more networks:
```
Comment (acuteenvy) [inline pages/common/docker-network.md:14]:
```suggestion
- Display detailed information about one or more networks:
```

File: pages/common/docker-service.md [hunk 16]
```diff
@@ -11,18 +11,18 @@
 
 `docker service create --name {{service_name}} {{image}}:{{tag}}`
 
-- Display detailed information of a space-separated list of services:
+- Display detailed information of one or more services:
```
Comment (acuteenvy) [inline pages/common/docker-service.md:14]:
```suggestion
- Display detailed information about one or more services:
```

File: pages/common/dust.md [hunk 17]
```diff
@@ -7,9 +7,9 @@
 
 `dust`
 
-- Display information for a space-separated list of directories:
+- Display information for one or more directories:
```
Comment (acuteenvy) [inline pages/common/dust.md:10]:
```suggestion
- Display information about one or more directories:
```

File: pages/common/git-cherry-pick.md [hunk 18]
```diff
@@ -14,7 +14,7 @@
 
 - Apply multiple (non-sequential) commits to the current branch:
 
-`git cherry-pick {{commit_1}} {{commit_2}}`
+`git cherry-pick {{commit_1 commit_2 ...}}`
```
Comment (acuteenvy) [inline pages/common/git-cherry-pick.md:17]:
```suggestion
`git cherry-pick {{commit1 commit2 ...}}`
```

File: pages/common/git-remote.md [hunk 19]
```diff
@@ -3,7 +3,7 @@
 > Manage set of tracked repositories ("remotes").
 > More information: <https://git-scm.com/docs/git-remote>.
 
-- Show a list of existing remotes, their names and URL:
+- List existing remotes with their names and URL:
```
Comment (acuteenvy) [inline pages/common/git-remote.md:6]:
```suggestion
- List existing remotes with their names and URLs:
```

File: pages/common/grunt.md [hunk 20]
```diff
@@ -7,9 +7,9 @@
 
 `grunt`
 
-- Run one or more specific space-separated task(s):
+- Run one or more task(s):
```
Comment (acuteenvy) [inline pages/common/grunt.md:10]:
```suggestion
- Run one or more tasks:
```

File: pages/common/ptargrep.md [hunk 21]
```diff
@@ -3,13 +3,9 @@
 > Find regular expression patterns in one or more tar archive files.
 > More information: <https://manned.org/ptargrep>.
 
-- Search for a pattern within a tar file:
+- Search for a pattern within one or more files:
```
Comment (acuteenvy) [inline pages/common/ptargrep.md:6]:
```suggestion
- Search for a pattern within one or more `tar` archives:
```

File: pages/common/qpdf.md [hunk 22]
```diff
@@ -5,24 +5,24 @@
...
 
 - Merge (concatenate) given pages from a list of PDF files and save the result as a new PDF:
 
-`qpdf --empty --pages {{file1.pdf}} {{1,6-8}} {{file2.pdf}} {{3,4,5}} -- {{output.pdf}}`
+`qpdf --empty --pages {{path/to/file1.pdf}} {{1,6-8}} {{path/to/file2.pdf}} {{3,4,5}} -- {{path/to/output.pdf}}`
 
 - Write each group of n pages to a separate output file with a given filename pattern:
 
-`qpdf --split-pages=n {{input.pdf}} {{out_%d.pdf}}`
+`qpdf --split-pages=n {{path/to/input.pdf}} {{path/to/out_%d.pdf}}`
```
Comment (acuteenvy) [inline pages/common/qpdf.md:20]:
```suggestion
`qpdf --split-pages={{n}} {{path/to/input.pdf}} {{path/to/out_%d.pdf}}`
```

File: pages/common/qpdf.md [hunk 23]
```diff
@@ -5,24 +5,24 @@
...
 
-`qpdf --empty --pages {{file1.pdf}} {{file2.pdf}} {{file3.pdf}} -- {{output.pdf}}`
+`qpdf --empty --pages {{path/to/file1.pdf file2.pdf ...}} -- {{path/to/output.pdf}}`
 
 - Merge (concatenate) given pages from a list of PDF files and save the result as a new PDF:
 
-`qpdf --empty --pages {{file1.pdf}} {{1,6-8}} {{file2.pdf}} {{3,4,5}} -- {{output.pdf}}`
+`qpdf --empty --pages {{path/to/file1.pdf}} {{1,6-8}} {{path/to/file2.pdf}} {{3,4,5}} -- {{path/to/output.pdf}}`
 
 - Write each group of n pages to a separate output file with a given filename pattern:
```
Comment (acuteenvy) [inline pages/common/qpdf.md:18]:
```suggestion
- Write each group of `n` pages to a separate output file with a given filename pattern:
```

File: pages/linux/auracle.md [hunk 24]
```diff
@@ -7,13 +7,13 @@
 
 `auracle search '{{regular_expression}}'`
 
-- Display package information for a space-separated list of AUR packages:
+- Display package information for one or more AUR packages:
```
Comment (acuteenvy) [inline pages/linux/auracle.md:10]:
```suggestion
- Display package information about one or more AUR packages:
```

File: pages/linux/auracle.md [hunk 25]
```diff
@@ -7,13 +7,13 @@
...
 
-`auracle info {{package1}} {{package2}}`
+`auracle info {{package1 package2 ...}}`
 
-- Display the `PKGBUILD` file (build information) for a space-separated list of AUR packages:
+- Display the `PKGBUILD` file (build information) for a one or more AUR packages:
```
Comment (acuteenvy) [inline pages/linux/auracle.md:14]:
```suggestion
- Display the `PKGBUILD` file (build information) of one or more AUR packages:
```

File: pages/linux/libreoffice.md [hunk 26]
```diff
@@ -3,17 +3,17 @@
...
-`libreoffice --view {{path/to/file1}} {{path/to/file2}}`
+`libreoffice --view {{path/to/file1 path/to/file2 ...}}`
 
-- Display the content of specific files:
+- Display the content of one or more files:
 
-`libreoffice --cat {{path/to/file1}} {{path/to/file2}}`
+`libreoffice --cat {{path/to/file1 path/to/file2 ...}}`
 
 - Print files to a specific printer:
```
Comment (acuteenvy) [inline pages/linux/libreoffice.md:14]:
```suggestion
- Print files using a specific printer:
```

File: pages/linux/scontrol.md [hunk 27]
```diff
@@ -9,16 +9,16 @@
 
 - Suspend a comma-separated list of running jobs:
 
-`scontrol suspend {{job_id}}`
+`scontrol suspend {{job_id1,job2,...}}`
```
Comment (acuteenvy) [inline pages/linux/scontrol.md:12]:
```suggestion
`scontrol suspend {{job_id1,job_id2,...}}`
```

File: pages/linux/scontrol.md [hunk 28]
```diff
@@ -9,16 +9,16 @@
...
 
 - Resume a comma-separated list of suspended jobs:
 
-`scontrol resume {{job_id}}`
+`scontrol resume {{job_id1,job2,...}}`
```
Comment (acuteenvy) [inline pages/linux/scontrol.md:16]:
```suggestion
`scontrol resume {{job_id1,job_id2,...}}`
```

File: pages/linux/scontrol.md [hunk 29]
```diff
@@ -9,16 +9,16 @@
...
 
 - Hold a comma-separated list of queued jobs (Use `release` command to permit the jobs to be scheduled):
 
-`scontrol hold {{job_id}}`
+`scontrol hold {{job_id1,job2,...}}`
```
Comment (acuteenvy) [inline pages/linux/scontrol.md:20]:
```suggestion
`scontrol hold {{job_id1,job_id2,...}}`
```

File: pages/linux/scontrol.md [hunk 30]
```diff
@@ -9,16 +9,16 @@
...
 
 - Release a comma-separated list of suspended job:
 
-`scontrol release {{job_id}}`
+`scontrol release {{job_id1,job2,...}}`
```
Comment (acuteenvy) [inline pages/linux/scontrol.md:24]:
```suggestion
`scontrol release {{job_id1,job_id2,...}}`
```

File: pages/windows/del.md [hunk 31]
```diff
@@ -8,9 +8,9 @@
 
 `tldr remove-item`
 
-- Delete one or more space-separated files or patterns:
+- Delete one or more files or patterns:
 
-`del {{file_pattern}}`
+`del {{file_pattern file_pattern}}`
```
Comment (acuteenvy) [inline pages/windows/del.md:13]:
```suggestion
`del {{file_pattern1 file_pattern2}}`
```

File: pages/windows/findstr.md [hunk 32]
```diff
@@ -3,34 +3,34 @@
 > Find specified text within one or more files.
 > More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/findstr>.
 
-- Find space-separated string(s) in all files:
+- Find one or more in all files:
```
Comment (acuteenvy) [inline pages/windows/findstr.md:6]:
```suggestion
- Find one or more strings in all files:
```

File: pages/windows/findstr.md [hunk 33]
```diff
@@ -3,34 +3,34 @@
...
 
-`findstr "{{query}}" *`
+`findstr "{{string1 string2 ...}}" *`
 
-- Find space-separated string(s) in a piped command's output:
+- Find one or more string(s) in a piped command's output:
```
Comment (acuteenvy) [inline pages/windows/findstr.md:10]:
```suggestion
- Find one or more strings in a piped command's output:
```

File: pages/windows/findstr.md [hunk 34]
```diff
@@ -3,34 +3,34 @@
...
 
-`{{dir}} | findstr "{{query}}"`
+`{{dir}} | findstr "{{string1 string2 ...}}"`
 
-- Find space-separated string(s) in all files recur[s]ively:
+- Find one or more string(s) in all files recur[s]ively:
```
Comment (acuteenvy) [inline pages/windows/findstr.md:14]:
```suggestion
- Find one or more strings in all files recur[s]ively:
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@sebastiaanspeck `{{path/to/output.opus}}`, etc generally suggest that only a file type is supported, so I would suggest staying consistent with other placeholders if there are multiple options then we use something like `.ext` (and mention in the description the support extensions, default ones.)

**Review by kbdharun — APPROVED** [review]

**Pull request merged**